tests/flink-it/src/test/scala/org/apache/celeborn/tests/flink/WordCountTest.scala

```diff
@@ -70,7 +69,6 @@ class WordCountTest extends AnyFunSuite with Logging with MiniClusterFeature
     val env = StreamExecutionEnvironment.createLocalEnvironmentWithWebUI(configuration)
     env.getConfig.setExecutionMode(ExecutionMode.BATCH)
     env.getConfig.setParallelism(parallelism)
-    env.getConfig.setDefaultInputDependencyConstraint(InputDependencyConstraint.ALL)
```

Comment:
[FLINK-20619][runtime] Remove unused InputDependencyConstraint
https://issues.apache.org/jira/browse/FLINK-20619
Fix Version/s:1.13.0
> It is no longer needed since both legacy scheduler and lazy-from-sources are removed.